core/src/main/java/com/microsoft/applicationinsights/internal/shutdown/SDKShutdownActivity.java

```diff
@@ -51,10 +53,16 @@
     private static class SDKShutdownAction implements Runnable {
         private boolean stopped = false;
 
+        private final Map<TelemetryChannel, Boolean> channels = new WeakHashMap<>();
+        @Deprecated
         private final List<ChannelFetcher> fetchers = new ArrayList<ChannelFetcher>();
```

Comment:
We can remove this since it's private.

Reply:
Let's keep `fetchers` list for now just in the (very unlikely) case someone is using it.